### ISSUE TITLE
The model and loaded state dict do not match exactly.

### ISSUE THREAD

**Issue by dleam** [aliyun/conditional-lane-detection#1]
Hi, I tried to use the given pretrained model for testing, however, I got the following results:

python tools/condlanenet/culane/test_culane.py configs/condlanenet/culane/culane_small_test.py history/official/culane_small.pth
The model and loaded state dict do not match exactly

unexpected key in source state_dict: bbox_head.reg_branch.0.conv.weight, bbox_head.reg_branch.0.bn.weight, bbox_head.reg_branch.0.bn.bias, bbox_head.reg_branch.0.bn.running_mean, bbox_head.reg_branch.0.bn.running_var, bbox_head.reg_branch.0.bn.num_batches_tracked, bbox_head.reg_branch.1.conv.weight, bbox_head.reg_branch.1.bn.weight, bbox_head.reg_branch.1.bn.bias, bbox_head.reg_branch.1.bn.running_mean, bbox_head.reg_branch.1.bn.running_var, bbox_head.reg_branch.1.bn.num_batches_tracked, bbox_head.reg_branch.2.conv.weight, bbox_head.reg_branch.2.bn.weight, bbox_head.reg_branch.2.bn.bias, bbox_head.reg_branch.2.bn.running_mean, bbox_head.reg_branch.2.bn.running_var, bbox_head.reg_branch.2.bn.num_batches_tracked

completed: 0, elapsed: 0s% 
Is there something wrong with the pretrained models or the codes?

**Comment by hustllz** [comment]
This is because some extra unnecessary layers are added during training. After finishing the code, these unnecessary layers were removed. These additional parameters will not have any impact on the accuracy and speed of the test.

**Comment by foolhard** [comment]
I met the exact same issue. Do you have any suggestion? 

python tools/condlanenet/culane/test_culane.py configs/condlanenet/culane/culane_small_test.py ./culane_small.pth 
The model and loaded state dict do not match exactly

unexpected key in source state_dict: bbox_head.reg_branch.0.conv.weight, bbox_head.reg_branch.0.bn.weight, bbox_head.reg_branch.0.bn.bias, bbox_head.reg_branch.0.bn.running_mean, bbox_head.reg_branch.0.bn.running_var, bbox_head.reg_branch.0.bn.num_batches_tracked, bbox_head.reg_branch.1.conv.weight, bbox_head.reg_branch.1.bn.weight, bbox_head.reg_branch.1.bn.bias, bbox_head.reg_branch.1.bn.running_mean, bbox_head.reg_branch.1.bn.running_var, bbox_head.reg_branch.1.bn.num_batches_tracked, bbox_head.reg_branch.2.conv.weight, bbox_head.reg_branch.2.bn.weight, bbox_head.reg_branch.2.bn.bias, bbox_head.reg_branch.2.bn.running_mean, bbox_head.reg_branch.2.bn.running_var, bbox_head.reg_branch.2.bn.num_batches_tracked
